### PR TITLE
Channel Edits - Syncronizing Tags Across Multi-Channels

### PR DESCRIPTION
**Pull request by jkemp814**
The edits continue.

### CODE REVIEW

**Comment by jkemp814** [comment]
Guys, somehow the logos for Maverick Black Cinema got put in Love Nation and Wipe Xtra. Neither of switch I edited. Wasn't even close or near those channels. I have corrected it already.

File: channels/us.m3u [hunk 1]
```diff
@@ -2484,7 +2484,7 @@ https://dmr-midnightpulp-3-us.samsung.wurl.tv/playlist.m3u8
 https://dai.google.com/linear/hls/event/1fO2zbpBRyy6S5yve_fnaw/master.m3u8
 #EXTINF:-1 tvg-id="MidnightPulpTV.us" tvg-name="Midnight Pulp TV" tvg-country="US" tvg-language="English" tvg-logo="https://github.com/geonsey/Free2ViewTV/blob/master/images/logos/MidnightPulp_200x200.png?raw=true" group-title="",Midnight Pulp TV (1080p)
 https://midnightpulp-samsung.plex.wurl.com/manifest/playlist.m3u8
-#EXTINF:-1 tvg-id="MihanTV.us" tvg-name="Mihan TV" tvg-country="US" tvg-language="" tvg-logo="" group-title="",Mihan TV (720p)
+#EXTINF:-1 tvg-id="MihanTV.us" tvg-name="Mihan TV" tvg-country="US" tvg-language="Farsi" tvg-logo="https://i.imgur.com/E6zzyqZ.jpg" group-title="News",Mihan TV (720p)
```
Comment (Dum4G) [inline channels/us.m3u:2487]: tvg-language="Persian" according to iptv-org automation scripts

File: channels/us.m3u [hunk 2]
```diff
@@ -1902,25 +1902,25 @@ https://t06858-lh.akamaihd.net/i/t06858a_1@719164/index_1500000_av-p.m3u8
...
 https://ion-ion-plus-1.vizio.wurl.com/manifest/playlist_640x360_1000k.m3u8
 #EXTINF:-1 tvg-id="IonPlus.us" tvg-name="Ion Plus" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/d449jnv.png" group-title="Family",Ion Plus (1080p)
 https://ion-plus.samsung.wurl.com/manifest/playlist.m3u8
 #EXTINF:-1 tvg-id="IonQubo.us" tvg-name="Ion Qubo" tvg-country="US" tvg-language="English" tvg-logo="http://vignette2.wikia.nocookie.net/logopedia/images/5/50/Qubo_logo_2012.png/revision/latest?cb=20150915211649" group-title="Kids",Ion Qubo
 https://ion-qubo-1.vizio.wurl.com/manifest/playlist.m3u8
-#EXTINF:-1 tvg-id="KKPX.us" tvg-name="ion TV (KKPX)" tvg-country="US" tvg-language="English" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/2/28/Ion_logo.svg/1920px-Ion_logo.svg.png" group-title="Local",ion TV (KKPX)
+#EXTINF:-1 tvg-id="IonTVKKPX.us" tvg-name="Ion TV (KKPX)" tvg-country="US" tvg-language="English" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/thumb/2/28/Ion_logo.svg/1920px-Ion_logo.svg.png" group-title="Local",Ion TV (KKPX)
```
Comment (Dum4G) [inline channels/us.m3u:1923]: tvg-logo is so big I can make a poster on my wall

File: channels/us.m3u [hunk 3]
```diff
@@ -2536,7 +2536,7 @@ https://stitcheraws.unreel.me/wse-node02.powr.com/live/5bf2218bd5eeee0f5a40941b/
 https://dai2-xumo.fubo.tv/amagi_hls_data_xumo1212A-motorvisiontv/CDN/master.m3u8
 #EXTINF:-1 tvg-id="MovieKingdom.us" tvg-name="Movie Kingdom" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/pIuJqYk.png" group-title="Movies",Movie Kingdom
 https://a.jsrdn.com/broadcast/e9b4093a41/+0000/high/c.m3u8
-#EXTINF:-1 tvg-id="MovieMex.us" tvg-name="Movie Mex" tvg-country="US" tvg-language="Spanish" tvg-logo="" group-title="Movies",Movie Mex
+#EXTINF:-1 tvg-id="Moovimex.us" tvg-name="Moovimex" tvg-country="US" tvg-language="Spanish" tvg-logo="" group-title="Entertainment",Moovimex
```
Comment (Dum4G) [inline channels/us.m3u:2539]: Shouldn't it remain in Movies category and tvg-country be MX (Mexico?)

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: channels/us.m3u (hunk 3 above)
Comment (jkemp814) [reply channels/us.m3u:2539]: Well, according to the sites I read, they do narco series and other programming. I can't find a logo anywhere for it. That's how I learned about the other shows.

Moovimex: Moovimex is a collection of Mexican action, narco and comedy movies.
Pongalo NovelaClub: Pongalo NovelaClub offers hit telenovelas and TV series from throughout Latin America all in Spanish.

Comment (Dum4G) [reply channels/us.m3u:2539]: No need to revert anything if you can prove you're right. You can make tvg-country MX tho

Comment (jkemp814) [reply channels/us.m3u:2539]: I'm not sure on the country, I can change it to what you guys think is best. As you can see, I didn't edit that part.

**Review by Dum4G — APPROVED** [review]

**Comment by LaneSh4d0w** [comment]
Was away at that moment, so far so good from what I've seen.

**Pull request merged**